apps/antalmanac/src/stores/Schedules.ts

```diff
@@ -9,10 +9,20 @@ import {
     ScheduleUndoState,
     ShortCourseSchedule,
 } from './schedule.types';
-import { RepeatingCustomEvent } from '$components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog';
-import { combineSOCObjects, CourseInfo, getCourseInfo, queryWebsoc } from '$lib/helpers';
-
-const defaultColors = [blue[500], pink[500], purple[500], green[500], amber[500], deepPurple[500], deepOrange[500]];
+import {RepeatingCustomEvent} from '$components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog';
+import {combineSOCObjects, CourseInfo, getCourseInfo, queryWebsoc} from '$lib/helpers';
+import { openSnackbar } from '$actions/AppStoreActions';
+import AppStore from './AppStore';
+
+const defaultColors = [
```

Comment:
Here's a funny strat I came up with to load all the possible colors we have lol; it has black and white too tho, so you'd have to deal with them too
https://github.com/icssc/AntAlmanac/blob/aponia-monorepo/apps/client/src/stores/schedule/course.ts#L17

Reply:
These aren't all MUI colors though. Some of them are too close, and the current ordering specifically avoids close colors for as long as possible.